### ISSUE TITLE
Contact form validation

### ISSUE THREAD

**Issue by irbasnet** [SproutTechAus/sprouttech-website#41]
Contact form validation is only done in client side using java script (see in myjs.js).Server side validation willbe updated till tomorrow. @prakashkhadka if you have any idea (blocking scripts/spams ) then do message me.

Thanks
rajesh

**Comment by prakashkhadka** [comment]
I think we should limit the number of words or letter that user can enter in the form. Valid email should be made necessary to stop spammers.

**Comment by irbasnet** [comment]
Email validation is performed please see if its working

Sent on the go with Vodafone-------- Original message --------From: prakashkhadka notifications@github.com Date: 06/08/2015  11:07  (GMT+10:00) To: SproutTechAus/sprouttech-website sprouttech-website@noreply.github.com Cc: gitrazesh basnetrahzes@hotmail.com Subject: Re: [sprouttech-website] Contact form validation (#41) 
I think we should limit the number of words or letter that user can enter in the form. Valid email should be made necessary to stop spammers.

**Comment by prakashkhadka** [comment]
Yup. Its perfect. Email validation is fine but it reset the message when invalid email address is entered. I think, it should ask  to enter valid email id only.

**Comment by irbasnet** [comment]
So is it reaerting message field only or others too ?
Please check and raise issue in guthub thanks 

Sent on the go with Vodafone-------- Original message --------From: prakashkhadka notifications@github.com Date: 06/08/2015  11:19  (GMT+10:00) To: SproutTechAus/sprouttech-website sprouttech-website@noreply.github.com Cc: gitrazesh basnetrahzes@hotmail.com Subject: Re: [sprouttech-website] Contact form validation (#41) 
Yup. Its perfect. Email validation is fine but it reset the message when invalid email address is entered. I think, it should ask  to enter valid email id only.

**Comment by gitrazesh** [comment]
hi prakash ,
Please find the log in details for cpanel
Username: rajeshba
Password: 26WMlyz22qControl Panel URL: https://xjdz3.dailyrazor.com:2083
Date: Thu, 6 Aug 2015 11:23:38 +1000
Subject: Re: [sprouttech-website] Contact form validation (#41)
From: basnetrahzes@hotmail.com
To: reply@reply.github.com

So is it reaerting message field only or others too ?
Please check and raise issue in guthub thanks 


Sent on the go with Vodafone-------- Original message --------From: prakashkhadka <notifications@github.com> Date: 06/08/2015  11:19  (GMT+10:00) To: SproutTechAus/sprouttech-website <sprouttech-website@noreply.github.com> Cc: gitrazesh <basnetrahzes@hotmail.com> Subject: Re: [sprouttech-website] Contact form validation (#41) 
Yup. Its perfect. Email validation is fine but it reset the message when invalid email address is entered. I think, it should ask  to enter valid email id only.
